misk-aws/src/test/kotlin/misk/jobqueue/sqs/DockerSqs.kt

```diff
@@ -22,7 +23,7 @@ import wisp.logging.getLogger
 internal object DockerSqs : ExternalDependency {
 
   private val log = getLogger<DockerSqs>()
-  private const val clientPort = 4100
+  private const val clientPort = 9324
```

Comment:
Is this going to conflict with anything or does this Docker image just have a different exposed port?

Reply:
The latter, elastic mq's default port is just different from the older image